Resources/Prototypes/_Lavaland/Procedural/lavaland_ruins.yml

```
@@ -1,104 +1,126 @@
-  path: Maps/_Lavaland/Lavaland/ruin_felenid_lab.yml
-
-- type: lavalandRuin
-  id: MiningPost
-  name: Miner Outpost
-  path: Maps/_Lavaland/Lavaland/ruin_mining_outpost.yml
-  
-- type: lavalandRuin
-  id: HierophantArena
-  name: Hierophant
-  path: Maps/_Lavaland/Lavaland/ruin_hierophant_arena.yml
+  id: CommonRuin5x5 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)